newa/__init__.py

```diff
@@ -477,9 +479,93 @@ class Request(Cloneable, Serializable):
     id: str
     context: RecipeContext = field(factory=dict)
     environment: RecipeEnvironment = field(factory=dict)
+    compose: Optional[str] = None
     git_url: Optional[str] = None
     git_ref: Optional[str] = None
     tmt_path: Optional[str] = None
+    plan: Optional[str] = None
+
+    def fetch_details(self) -> None:
+        raise NotImplementedError
+
+    def generate_tf_exec_command(self) -> str:
+        cmdline_start = "NO_COLOR=yes testing-farm request --no-wait"
+        # check compose
+        if not self.compose:
+            print('ERROR: compose is not specified for the request')
```

Comment:
Shouldn't we raise an exception instead? It will end up that way anyway, `return ''` here leads to `raise Exception` on line 528.

Reply:
Oh, I forgot to update this one. Will fix it.

Reply:
fixed